asmcli/asmcli.sh

```diff
@@ -37,7 +37,7 @@ _CI_I_AM_A_TEST_ROBOT="${_CI_I_AM_A_TEST_ROBOT:=0}"; readonly _CI_I_AM_A_TEST_RO
 MAJOR="${MAJOR:=1}"; readonly MAJOR;
 MINOR="${MINOR:=15}"; readonly MINOR;
 POINT="${POINT:=7}"; readonly POINT;
-REV="${REV:=1}"; readonly REV;
+REV="${REV:=8}"; readonly REV;
 CONFIG_VER="${CONFIG_VER:="2"}"; readonly CONFIG_VER;
```

Comment:
CONFIG_VER should be 1 since this is a new rev.

Reply:
done